OverlayPlugin.Core/NetworkProcessors/LineEventStartPlayFinish.cs

```diff
@@ -0,0 +1,436 @@
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlay32 : IPacketStruct
+        {
+            EventPlayBase eventPlayBase;
+
+            const int eventParamsCount = 32;
+
+            fixed uint eventParams[eventParamsCount];
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlay64 : IPacketStruct
+        {
+            EventPlayBase eventPlayBase;
+
+            const int eventParamsCount = 64;
+
+            fixed uint eventParams[eventParamsCount];
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlay128 : IPacketStruct
+        {
+            EventPlayBase eventPlayBase;
+
+            const int eventParamsCount = 128;
+
+            fixed uint eventParams[eventParamsCount];
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        internal unsafe struct EventPlay255 : IPacketStruct
+        {
+            EventPlayBase eventPlayBase;
+
+            const int eventParamsCount = 255;
+
+            fixed uint eventParams[eventParamsCount];
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                fixed (uint* paramsPtr = eventParams)
+                {
+                    return eventPlayBase.ToString(paramsPtr, eventParamsCount);
+                }
+            }
+        }
+
+        [StructLayout(LayoutKind.Sequential, Pack = 1)]
+        public unsafe struct EventStart : IPacketStruct
+        {
+            uint actorID;
+            uint unknown1;
+            uint eventID;
+            byte param1;
+            byte param2;
+            ushort padding;
+            uint param3;
+            uint padding1;
+
+            public string ToString(long epoch, uint ActorID)
+            {
+                return $"S|{actorID:X8}|{unknown1:X8}|{eventID:X8}|{param1:X2}|{param2:X2}|{param3:X8}";
```

Comment:
One thing that I think may be problematic from a consumer standpoint is that the "static" (non-eventParam) fields are different between EventPlay*, EventStart, and EventFinish, which makes an accurate log def a little more challenging.  To standardize, what about adding (either empty-string or zero-filled) placeholders for `scene` and `flags`?

Also, just in terms of formatting, could we change `unknown1`, `eventID`, and `param1-3` to `:X` so they match `EventPlay*`?